### PR TITLE
Update phpdoc in AuthComponent

### PR DESCRIPTION
**Pull request by podliy16**

### CODE REVIEW

**Comment by dereuromark** [comment]
:+1:

File: src/Controller/Component/AuthComponent.php [hunk 1]
```diff
@@ -657,7 +657,7 @@ public function logout()
      * Get the current user from storage.
      *
      * @param string $key Field to retrieve. Leave null to get entire User record.
```
Comment (dereuromark) [inline src/Controller/Component/AuthComponent.php]: This should be `string|null` by the way.

**Comment by podliy16** [comment]
Done

**Comment by dereuromark** [comment]
Thank you! :)

**Pull request merged**